css/main.css

```diff
@@ -56,19 +61,177 @@ textarea {
    Author's custom styles
    ========================================================================== */
 
+ul {
+  list-style-type: none;
+  padding: 0;
+}
+
+.black {
+  color: #000;
+}
+
+body {
+  width: 100%;
+  height: 100%;
+}
+
+.main {
+  display: flex;
+  overflow: hidden;
+  font: normal 16px sans-serif;
+  width: 100%;
+  height: 100%;
+}
+
+.sidebar {
+  display: flex;
+  flex-direction: column;
+  background: #0e171d;
+  color: #fff;
+  width: 300px;
+  height: 100%;
+  float: left;
+}
+
+.logo-motorpool {
+  background-image: url("../img/logo.png");
+  background-repeat: no-repeat;
+  margin-left: 35px;
+  margin-top: 40px;
+  margin-bottom: 46px;
```

Comment:
Same here with the shorthand, I would probbbbbbably use `margin: 40px 0 46px 35px;` instead — when you set individual margins like this, it can look like you're overriding another rule that's set somewhere (in this case, I might assume that the margin-right is being set by another rule, and you're specifically trying to not touch it).

THIS IS MAYBE JUST A DOUG OPINION